README.md

```diff
@@ -9,97 +9,48 @@
   </a>
 </p>
 
-## Table of Contents
-* [What is eoAPI](#whatitis)
-* [Getting Started](#gettingstarted)
-* [Helm Installation](#helminstall)
-* [Default Configuration and Options](#options)
-* [Autoscaling / Monitoring / Observability](./docs/autoscaling.md)
-
-<a name="whatitis"/>
-
 ## What is eoAPI?
 
 [https://eoapi.dev/](https://eoapi.dev/)
 
-<a name="gettingstarted"/>
-
 ## Getting Started
 
+Make sure you have [helm](https://helm.sh/docs/intro/install/) installed on your machine.
+Additionally, you will need a cluster to deploy the eoAPI helm chart. This can be on a cloud provider, like AWS, GCP, or any other that supports Kubernetes. You can also run a local cluster using minikube.i
```

Comment:
```suggestion
Additionally, you will need a cluster to deploy the eoAPI helm chart. This can be on a cloud provider, like AWS, GCP, or any other that supports Kubernetes. You can also run a local cluster using minikube.
```